static/app/utils/discover/eventView.spec.tsx

```diff
@@ -503,6 +503,64 @@ describe('EventView.fromSavedQuery()', function () {
   });
 });
 
+describe('EventView.fromNewQueryWithPageFilters', function () {
```

Comment:
```suggestion
describe('EventView.fromNewQueryWithPageFilters()', function () {
```

Small nit, just to be consistent with the other tests!